DOCS-8223-Export-SAML (#77)
mulesoft/docs-access-management: modules/ROOT/pages/managing-users.adoc

+
. Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.

== Export SAML Metadata

> If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings. You can then upload this file to your identity provider to configure Anypoint Platform.

Reviewer: "service provider metadata": when two nouns are used as a single adjective to modify a third noun, place a hyphen between those two nouns. In this case, "service" and "provider" are being used as a single adjective to modify "metadata". So, "service provider metadata" should be "service-provider metadata". Otherwise, you get a noun string, and those can be difficult to make sense of.

> If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings. You can then upload this file to your identity provider to configure Anypoint Platform.

Reviewer: "an XML file of your SAML configuration settings": I would put the emphasis on the settings, like this: "to download your SAML configuration settings in an XML file". Otherwise, I wait until the end of the sentence to read why I'd want to download this file.

Author: ```suggestion
If your identity provider supports uploading service-provider metadata, you can click the *Anypoint service provider metadata* link to download your SAML configuration settings in an XML file. You can then upload this file to your identity provider to configure Anypoint Platform.
```

> If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings to send to your identity provider. The identity provider can then upload these configuration settings to connect to your Anypoint Platform community.

Author: "Anypoint Platform community" seems kind of weird. I was emulating the SF language. Is this a thing, @santro92 ?

Reviewer 2: Nope keeping it as *to connect to Anypoint Platform* is enough

Author: ```suggestion
If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings to send to your identity provider. The identity provider can then upload these configuration settings to connect to Anypoint Platform.
```

Reviewer 3: What about this:

"If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings. Next, upload this file to your identity provider to configure Anypoint Platform."

Author: ```suggestion
If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings. You can then upload this file to your identity provider to configure Anypoint Platform.
```
